build/yaml/webhook/service.yaml

```diff
@@ -18,3 +18,4 @@ spec:
       targetPort: 9981
   selector:
     component: nsx-ncp
+    role: master
```

Comment:
Is there any way to identify it is nsx-operator master, as it could be a different Pod for ncp master.

Reply:
I suppose it is the clearest, any problem for a different Pod for ncp master?

Reply:
Since all tests passed, added a comment to this label.